hotungkhanh/jetedge
# feat: add more startTimes

Example test instruction:
1. Upload and send any valid large timetable problem to backend using frontend (i.e. 5000+ students, 50+ units)
2. Check Gantt Chart to see time allocated ranges from 8am to 6pm with 30 minutes time step
